client/src/app/pages/sbom-details/packages-by-sbom.tsx

```diff
@@ -220,17 +220,12 @@ export const PackagesBySbom: React.FC<PackagesProps> = ({ sbomId }) => {
                       modifier="breakWord"
                       {...getTdProps({
                         columnKey: "licenses",
-                        isCompoundExpandToggle: item.licenses.length > 1,
+                        isCompoundExpandToggle: item.licenses.length > 0,
                         item: item,
                         rowIndex,
                       })}
```

Comment:
There is no need to add the compound properties now. This block of code can become:

```ts
{...getTdProps({
  columnKey: "licenses",
})}
```